server/tables/tables.go

```diff
@@ -243,6 +243,25 @@ func (t *Token) TableName() string {
 	return "Tokens"
 }
 
+type APIKey struct {
+	Model
+
+	APIKeyID     string `gorm:"primary_key"`
+	GroupGroupID string `gorm:"index:group_id_index"`
```

Comment:
indexes in gorm are (annoyingly) in the global namespace by default. So I think we should prefer to prefix index names with the table names to keep things sane.

So can you change group_id_index and value_index to...

api_key_group_id_index
api_key_value_index

Reply:
Ah, unfortunate. Done